packages/web-components/@groupby/elements-search-box/test/unit/search-box.test.ts

```diff
@@ -101,10 +101,18 @@ describe('SearchBox Component', () => {
   });
 
   describe('emitSearchEvent', () => {
-    it('should dispatch a search request event with empty area and collection', () => {
-      const query = searchbox.value = 'a';
+    const area = 'some-area';
+    const collection = 'some-collection';
+    let origin;
```

Comment:
If this is customized once, and otherwise used as an assertion in the other times, I don't think we need this variable. I think we can have `origin: 'search'` in the assertions and for the one case where it's customized, make it a local variable.